pytorch_lightning/callbacks/progress.py

```diff
@@ -340,8 +343,9 @@ def on_train_batch_end(self, trainer, pl_module, batch, batch_idx, dataloader_id
 
     def on_validation_start(self, trainer, pl_module):
         super().on_validation_start(trainer, pl_module)
-        self.val_progress_bar = self.init_validation_tqdm()
-        self.val_progress_bar.total = convert_inf(self.total_val_batches)
+        if self._sanity_check_completed is True:
```

Comment:
these flags aren't needed...

we already have:

```python
if trainer.running_sanity_check:
```

Reply:
didn't know about this. Updated!